resources/config_auth_generic_oauth.rb

```diff
@@ -67,3 +67,7 @@
     accumulator_config(:set, rp.to_s.delete_prefix('oauth_'), new_resource.send(rp))
   end
 end
+
+def resource_config_path_override
+  %w(auth.generic_oauth)
```

Comment:
Why are we using an array here rather than just a string?

Reply:
This is correct as the code expects an Array so it can deal with nested paths.